connectors/sync_job_runner.py

```diff
@@ -151,6 +151,13 @@ async def execute(self):
                 self._data_source_framework_config()
             )
 
+            # Only full syncs restart from a checkpoint that's stored in a sync job
+            if self.sync_job.sync_cursor and self.sync_job.job_type == JobType.FULL:
+                self.sync_job.log_debug(
```

Comment:
```suggestion
                self.sync_job.log_info(
```
loosely-held suggestion. Seems low-noise, but valuable.